src/subapps/admin/components/Settings/PermissionsAclsSubView.tsx

```diff
@@ -21,6 +21,7 @@ type DataType = {
   create?: boolean;
   query?: boolean;
   children?: DataType[];
+  realm?: string;
```

Comment:
I think it would make sense to create another TypeScript type for `children` that excludes `realm` to make this more cleaner. I'm thinking something along the lines of `type ChildType = Omit<DataType, 'realm'>`. We can ofcourse exclude more keys too if it makes sense.

This way we can make the code below clearer too by using typeguards to check if the row is for a parent or child.